tests/generate_small_sample.py

```diff
@@ -0,0 +1,133 @@
+from test_main import TestFiles
+from pathlib import Path
+import hashlib
+
+from pysource_minimize import minimize
+import sys
+import subprocess as sp
+import textwrap
+import os
+import linecache
+from executing import Source
+from multiprocessing import Pool,get_context
+import tempfile
+import hashlib
+import time
+
+last_samples_dir = Path(__file__).parent / "last_samples"
+last_samples_dir.mkdir(exist_ok=True)
+samples_dir = Path(__file__).parent / "samples"
+
+
+# big_samples = (Path(__file__).parent / "samples").glob("*.py")
+# big_samples = list(Path.home().rglob("*.py"))
+small_samples = Path(__file__).parent / "small_samples"
+
+
+def big_samples():
+    yield from last_samples_dir.rglob("*.py")
+    yield from samples_dir.rglob("*.py")
+
+    hashes = set()
+    root = Path.home()
+    # root = Path(__file__).parent / "samples"
```

Comment:
.

Reply:
I think this means remove the commented code

Reply:
Yes. If the idea is to uncomment the code under certain conditions when developing, add a comment explaining when that's a good idea.